rustler_mix/README.md

```diff
@@ -15,65 +14,22 @@ This package is available on [`hex.pm`](https://hex.pm/packages/rustler). To ins
 
 ```elixir
 def deps do
-  [{:rustler, "~> 0.21.0"}]
+  [{:rustler, "~> 0.21.0", runtime: false}]
 end
 ```
 
 Then,
 
 1. Run `mix deps.get` to fetch the dependency.
-2. Run `mix rustler.new` and follow the instructions to generate the boilerplate for your NIF.
-3. Enable the `:rustler` mix compiler by adding `compilers: [:rustler] ++ Mix.compilers(),` to the `project` section of your `mix.exs`.
-4. Add a configuration entry to the `rustler_crates` section of your `mix.exs`. [See below](#crate-configuration).
-5. Load the NIF in your program. [See below](#loading-the-nif).
+1. Run `mix rustler.new` and follow the instructions to generate the boilerplate for your NIF.
+1. Load the NIF in your program. [See below](#loading-the-nif).
 
 ## Crate configuration
 
-The `rustler_crates` configuration is a keyword list mapping the crate name (an atom) to the NIF configuration (another keyword list).
-The NIF configuration may contain the following entries:
+The Rust crate compilation can be controlled via Mix compile-time configuration in `config/config.exs`.
+See [configuration options](https://hexdocs.pm/rustler/Rustler.html#module-configuration-options) for more
```

Comment:
The anchor is broken.

Reply:
Yeah, it will be until this version is published.